lms/templates/ux/reference/course-skeleton.html

```diff
@@ -64,8 +64,30 @@ <h2 class="hd hd-2 page-title">Skeleton Page</h2>
         </div>
     </header>
     <div class="page-content">
-        <h3>This is where the page content belongs</h3>
-        <p>Useful stuff goes here</p>
+        <div class="layout layout-1t2t">
+            <aside class="layout-col layout-col-a" role="complementary" aria-label="Navigation">
```

Comment:
See my earlier comment about this. Let's make it a `section` with a heading as the first child. The `section` should use `aria-labelledby` and reference that newly added heading. We can also remove the role here.